airbyte-webapp/.stylelintignore

```diff
@@ -0,0 +1 @@
+coverage/**/*.css
```

Comment:
nit of all nits, but could you put a final newline in this file? `.editorconfig` is agnostic except for python files, where `insert_final_newline = true` is explicitly set, but our files nearly universally have them. Perhaps most importantly, though, is that I think github's big red "no final newline here" icon is kind of ugly :shrug: :sweat_smile:

Reply:
Is there a reason to scope this specifically to `*.css` files? IIUC, everything in `coverage/` is autogenerated, so why not ignore `coverage/**/*`?

Reply:
I used the inline editor, I think it got the newline